### PR TITLE
Add BFD config to subinterface and neighbor

### PR DESCRIPTION
**Pull request by sojindal**

### CODE REVIEW

File: sonic_api.yaml [hunk 1]
```diff
@@ -2274,6 +2274,35 @@ definitions:
       ip_addr:
         type: string
         description: IP address of directly attached device to the switch
+  BfdEntry:
+    type: object
+    properties:
+      if_name:
```
Comment (lguohan) [inline sonic_api.yaml:2280]: can this be a ref?

File: sonic_api.yaml [hunk 2]
```diff
@@ -2274,6 +2274,35 @@ definitions:
...
+        type: string
+      tx_interval:
+        type: integer
+        description: interval that this system will use between transmission of control packets
```
Comment (lguohan) [inline sonic_api.yaml:2284]: can this have default value? the same comments go to other parameters.

File: sonic_api.yaml [hunk 3]
```diff
@@ -2274,6 +2274,35 @@ definitions:
...
+      rx_interval:
+        type: integer
+        descrition: minimum interval this system is capable of receiving control packets
+      multiplier:
+        type: integer
+        description: detection multiplier to detect packet loss for incoming packets
+      echo_interval:
+        type: integer
+        description: minimum interval for transmission of echo packets
+      echo_mode:
```
Comment (lguohan) [inline sonic_api.yaml:2294]: default value?

File: sonic_api.yaml [hunk 4]
```diff
@@ -2274,6 +2274,35 @@ definitions:
...
+        type: boolean
+        description: enables or disables echo packets
+      passive_mode:
```
Comment (lguohan) [inline sonic_api.yaml:2297]: default value?

File: sonic_api.yaml (hunk 2 above)
Comment (sojindal) [reply sonic_api.yaml]: Added.

File: sonic_api.yaml (hunk 3 above)
Comment (sojindal) [reply sonic_api.yaml:2300]: Added.

File: sonic_api.yaml (hunk 4 above)
Comment (sojindal) [reply sonic_api.yaml:2304]: Added.

File: sonic_api.yaml (hunk 1 above)
Comment (sojindal) [reply sonic_api.yaml:2280]: BfdEntry is added as ref per neighbor. Should it be something else?

Comment (lguohan) [reply sonic_api.yaml:2280]: i mean ref to the interface such as ip interface?

**Comment by lguohan** [comment]
other parts look good.

File: sonic_api.yaml (hunk 1 above)
Comment (sojindal) [reply sonic_api.yaml:2280]: 'if_name' name should be sufficient to identify the interface right?

**Pull request merged**